### PR TITLE
Web Interface

### PR DESCRIPTION
**Pull request by RobinDaugherty**
Add a Rack-compatible web interface that shows the currently-active revision of each Ember-CLI app and allows rollback to be performed directly from the UI.

* [x] Web tests.
* [x] WebHelper tests.
* [x] Ability to specify revision based on query param.
* [x] Link to specific revision using query param.
* [x] Roll back by inserting the revision at the top of the list.
* [x] Give credit to sidekiq/web in the README.

### CODE REVIEW

**Comment by Geesu** [comment]
I wish this was a gem.

**Comment by RobinDaugherty** [comment]
@Geesu when you comment on a commit, it doesn't add a line comment and it doesn't send it to Slack.

**Comment by RobinDaugherty** [comment]
This is a gem. I'm going to bump the version to 1.0.0 and publish it.

File: README.md [hunk 1]
```diff
@@ -12,48 +12,103 @@ the index page contents in Redis, and this gem allows access to it so that it ca
...
 ```
 
 ### Getting the contents of a deployed file
 
 ```ruby
-  EmberCliDeployRedis.latest_revision.contents_of('index.html')
+begin
+  EmberCliDeployRedis.active_revision.contents_of('index.html')
+rescue EmberCliDeployRedis::FileMissingFromDeployment
+  # Don't rescue this unless you have a fix for it. This is a real exception!
```
Comment (Geesu) [inline README.md]: Should you not re-raise the exception? Or is this a TODO?

**Comment by Geesu** [comment]
LGTM besides the one comment

**Pull request merged**